beacon-chain/rpc/prysm/v1alpha1/validator/status.go

```diff
@@ -110,82 +111,70 @@ func (vs *Server) CheckDoppelGanger(ctx context.Context, req *ethpb.DoppelGanger
 		return nil, status.Error(codes.Internal, "Could not get head state")
 	}
 
-	currEpoch := slots.ToEpoch(headState.Slot())
-	isRecent, resp := checkValidatorsAreRecent(currEpoch, req)
+	// Return early if we are in phase0.
+	if headState.Version() == version.Phase0 {
+		return nil, status.Errorf(codes.FailedPrecondition, "Doppelganger test not available for Phase0")
```

Comment:
This will error our on the validator's side and it will crash. Is there anyway we can make this not happen ?

Reply:
After more thought, how about not returning an error and just exiting early ? Maybe log a warning, this would allow phase 0 users to not crash when running the check.